ui/packages/catalog-ui-search/src/main/webapp/component/singletons/metacard-definitions.js

```diff
@@ -46,6 +46,40 @@ define([
             return result;
         }, {});
     }
+    const metacardStartingTypes = {
```

Comment:
are these not defined anywhere else?

Reply:
They were defined [here](https://github.com/codice/ddf/pull/3192/files/866125c10d0557099af2840668d4ccd59dc6e88d#diff-1e9738864a4dec24e514370fec010aa5L152) but they got moved to this const declaration